cli/test/cli-test-utils.ts

```diff
@@ -1,13 +1,31 @@
-import { BaseOptionsDto } from 'src/cores/dto/base-options-dto';
 import fs from 'node:fs';
 import path from 'node:path';
+import { ImmichApi } from '../src/services/api.service';
 
 export const TEST_CONFIG_DIR = '/tmp/immich/';
 export const TEST_AUTH_FILE = path.join(TEST_CONFIG_DIR, 'auth.yml');
 export const TEST_IMMICH_INSTANCE_URL = 'https://test/api';
 export const TEST_IMMICH_API_KEY = 'pNussssKSYo5WasdgalvKJ1n9kdvaasdfbluPg';
 
-export const CLI_BASE_OPTIONS: BaseOptionsDto = { config: TEST_CONFIG_DIR };
+export const CLI_BASE_OPTIONS = { config: TEST_CONFIG_DIR };
+
+export const setup = async () => {
+  const api = new ImmichApi(process.env.IMMICH_INSTANCE_URL as string, '');
+  await api.authenticationApi.signUpAdmin({
+    signUpDto: { email: 'cli@immich.app', password: 'password', name: 'Administrator' },
+  });
+  const { data: admin } = await api.authenticationApi.login({
+    loginCredentialDto: { email: 'cli@immich.app', password: 'password' },
+  });
+  const { data: apiKey } = await api.keyApi.createApiKey(
+    { aPIKeyCreateDto: { name: 'CLI Test' } },
```

Comment:
the capitalization looks off here

Reply:
Haha yeah. This is the auto generated code from open API generator. I think if we changed the dto on the server from APIKey to ApiKey it would generate more how you would expect it to.

Reply:
Based on [this](https://stackoverflow.com/questions/15526107/acronyms-in-camelcase) SO post, It seems style guidelines for acronyms lean toward using the normal casing instead of capitalizing. So `XmlHttpRequest`, not `XMLHTTPRequest`